lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -234,6 +235,12 @@ def resource_objects_for(serializers)
         [@primary, @included]
       end
 
+      def paginate?
+        serialization_context &&
+          serializer.respond_to?(:paginated?) &&
```

Comment:
Actually I'd rather not use `try` in the end as it might somehow implies that the serializer might be `nil`.